source/src/ports/generic_networkhandler.c

```diff
@@ -1093,6 +1093,7 @@ void CheckAndHandleConsumingUdpSocket(void) {
         int error_code = GetSocketErrorNumber();
         char *error_message = GetErrorMessage(error_code);
```

Comment:
This line should just be moved further down. The message is not needed in this case.